coop_account_check_deposit/models/account_account.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    bank_acc_number = fields.Char(readonly=True, store=True)
```

Comment:
Is this required in `12.0`?
I believe in `12.0` the bank_acc_number field is already present, and it's a related field to the bank account.

If there's no real purpose for this, I prefer to remove this file

Reply:
this field also in 9.0, but this module replace the related behaviour and change to char field.

Reply:
But it's already a Char field in 12.0.
The only thing that this is really doing is making it stored, but I see no point.

Can you remove this file?

Reply:
Okay i will remove it